lib/matplotlib/axes/_axes.py

```diff
@@ -3010,9 +3024,20 @@ def pie(self, x, explode=None, labels=None, colors=None,
             raise ValueError("Wedge sizes 'x' must be non negative values")
 
         sx = x.sum()
-        if sx > 1:
-            x = x / sx
 
+        if normalize is None:
+            if sx < 1:
+                cbook.warn_deprecated(
```

Comment:
Per our discussion on Gitter, we might want to consider having this retain the old behavior instead for at least one point release while warning. Currently, this changes the behavior immediately.